README.md

```diff
@@ -170,6 +170,124 @@ TODO
 ## Development
 TODO
 
+## Guides
+
+### Different roles in the same pod
+
+You can use different IAM roles in different containers within the same pod, but
+there is additional setup and configuration that you'll have to modify your pod
```

Comment:
there is additional setup and configuration _and_ you'll have to modify your pod